### PR TITLE
New phase banner for prod

### PR DESCRIPTION
**Pull request by jinnashravan**
Reference: https://sgs-ddt-01-96c924f9e494.herokuapp.com/registration/current/phase-banners

### CODE REVIEW

File: request_a_govuk_domain/request/templates/base.html [hunk 1]
```diff
@@ -66,18 +66,20 @@
 
   {% block phase_banner %}
     <div class="govuk-phase-banner govuk-width-container">
-      <p class="govuk-phase-banner__content">
+      <div class="govuk-phase-banner__content">
```
Comment (maxf) [inline request_a_govuk_domain/request/templates/base.html:69]: Why this change?

File: request_a_govuk_domain/request/templates/base.html [hunk 2]
```diff
@@ -66,18 +66,20 @@
...
         <strong class="govuk-tag {{ PHASE_CLASS }} govuk-phase-banner__content__tag">
           {% block phase_banner__tag %}
               {{PHASE_HEADER}}
           {% endblock %}
         </strong>
         <span class="govuk-phase-banner__text">
           {% block phase_banner__text %}
+          {% autoescape off %}
```
Comment (maxf) [inline request_a_govuk_domain/request/templates/base.html:77]: Why did you add this? Can you explain in the commit message?
Also: fix indentation

File: request_a_govuk_domain/request/templates/base.html (hunk 1 above)
Comment (jinnashravan) [reply request_a_govuk_domain/request/templates/base.html:69]: > Why this change?

Wanted to keep HEADER and CONTENT of banner in one line.

File: request_a_govuk_domain/request/templates/base.html (hunk 2 above)
Comment (jinnashravan) [reply request_a_govuk_domain/request/templates/base.html]: > Why did you add this? Can you explain in the commit message? Also: fix indentation
Done, fixed indentation and added the autoescape explanation.

Comment (jinnashravan) [reply request_a_govuk_domain/request/templates/base.html]: > Why did you add this? Can you explain in the commit message? Also: fix indentation

Done, fixed Indentation and added the autoescape explanation.

**Review by maxf — APPROVED** [review]
